app/views/administrate/application/_icons.erb

```diff
@@ -0,0 +1,15 @@
+<svg xmlns="http://www.w3.org/2000/svg" style="display: none;">
+  <symbol id="icon-eyeglass" viewBox="0 0 48 48">
+    <path d="M27.885 32.515c-2.864 1.966-6.333 3.116-10.07 3.116C7.976 35.63 0 27.656 0 17.817 0 7.976 7.976 0 17.816 0S35.63 7.976 35.63 17.816c0 3.736-1.15 7.205-3.115 10.07l14.53 14.53c1.278 1.277 1.275 3.352 0 4.628-1.28 1.278-3.353 1.278-4.63 0l-14.53-14.53zm-10.07-3.736c6.056 0 10.964-4.91 10.964-10.964 0-6.055-4.91-10.964-10.964-10.964-6.055 0-10.964 4.91-10.964 10.964 0 6.055 4.91 10.963 10.964 10.963z" />
+  </symbol>
+
+  <symbol id="icon-cancel" viewBox="0 0 48 48">
+    <g fill="#D8D8D8" fill-rule="evenodd">
```

Comment:
nvm, looks like you took care of it here: https://github.com/thoughtbot/administrate/pull/881/files#diff-55faa153a14542fdc12f2cb7a008afc7R3